sample/host.json

```diff
@@ -1,35 +1,35 @@
 {
-    "watchDirectories": [ "Shared", "Test" ],
-    "healthMonitor": {
-        "enabled": true,
-        "healthCheckInterval": "00:00:10",
-        "healthCheckWindow": "00:02:00",
-        "healthCheckThreshold": 6,
-        "counterThreshold": 0.80
+  "watchDirectories": [ "Shared", "Test" ],
+  "healthMonitor": {
+    "enabled": true,
+    "healthCheckInterval": "00:00:10",
+    "healthCheckWindow": "00:02:00",
+    "healthCheckThreshold": 6,
+    "counterThreshold": 0.80
+  },
+  "functionTimeout": "00:05:00",
+  "logger": {
+    "fileLoggingMode": "always"
+  },
+  "swagger": {
+    "enabled": true
+  },
+  "extensions": {
+    "sendGrid": {
+      "from": "Azure Functions <samples@functions.com>"
     },
-    "functionTimeout": "00:05:00",
-    "logger": {
-        "fileLoggingMode": "always"
+    "http": {
+      "routePrefix": "api",
+      "maxConcurrentRequests": 5,
+      "maxOutstandingRequests": 30
     },
-    "swagger": {
-        "enabled": true
+    "queues": {
+      "visibilityTimeout": "00:00:10"
     },
-    "extensions": {
```

Comment:
Why the changes? These had all been move under the `extensions`, which is where they should be.

Reply:
Will undo. Did not intend to make this change.